packages/style-engine/src/styles/index.ts

```diff
@@ -17,4 +18,5 @@ export const styleDefinitions = [
 	...spacing,
 	...typography,
 	...shadow,
+	...media,
```

Comment:
At first I questioned the `media` top-level grouping, wondering if we could use `background`, given it has a lot of [related properties](https://developer.mozilla.org/en-US/docs/Web/CSS/background). 

I think it's a good decision however since most, probably all, of those properties manipulate a background element most likely to be an image, and then we'd also have a neat home for stuff like [object-fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) and anything related to [replaced elements](https://developer.mozilla.org/en-US/docs/Web/CSS/Replaced_element) (videos, canvas, etc)